kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/WatchExample.java

```diff
@@ -67,7 +75,106 @@ public void onClose(KubernetesClientException e) {
+      .addToContainers(new ContainerBuilder().withName("nginx").withImage("nginx").build())
+      .endSpec()
+      .build();
+
+    client.pods().inNamespace("ns1").create(pod1);
+
+    Thread.sleep(5000);
+    Watch watch = client.pods().inNamespace("ns1")
+      .withLabels(new HashMap<String, String>() {{ put("test", "watch");}})
+      .watch(new Watcher<Pod>() {
+        @Override
+        public void eventReceived(Action action, Pod resource) {
+          switch (action) {
+            case DELETED:
+              System.out.println("DELETED  " +  resource.getMetadata().getName());
+              break;
+            case ADDED:
+              System.out.println("ADDED  " + resource.getMetadata().getName());
+              break;
+            case MODIFIED:
+              System.out.println("EDITED  " + resource.getMetadata().getName());
+              break;
+            default:
+              throw new AssertionFailedError(action.toString());
+          }
+        }
+
+        @Override
+        public void onClose(KubernetesClientException cause) {
+          System.out.println("CLOSSSSSINGGG");
+        }
+      });
+
+    Pod pod2 = new PodBuilder()
+      .withNewMetadata()
+      .withName("pod2")
+      .endMetadata()
+      .withNewSpec()
+      .addToContainers(new ContainerBuilder().withName("nginx").withImage("nginx").build())
+      .endSpec()
+      .build();
+
+    client.pods().inNamespace("ns1").create(pod2);
+
+    //pod2 = client.pods().inNamespace("ns1").withName("pod2").get();
+    //pod2.getMetadata().setLabels(new HashMap<String, String>() {{ put("test", "watch"); }});
+
+//    System.out.println(pod2.getMetadata());
+    Thread.sleep(10000);
+
+    System.out.println("ADDING THE TEST LABEL");
+    client.pods().inNamespace("ns1").withName("pod2").edit()
+      .editMetadata().addToLabels("test", "watch").endMetadata()
+      .done();
+
+    //System.out.println("EXECUTING REPLACE");
+   //pod2.getSpec().getContainers().get(0).setImage("redis");
+
+
+
+    //client.pods().inNamespace("ns1").withName("pod2").replace(pod2);
+    //client.pods().inNamespace("ns1").withName("pod2")
+      //.patch(pod2);
+
+    Thread.sleep(10000);
+
+    //pod2 = client.pods().inNamespace("ns1").withName("pod2").get();
+
+    System.out.println("REMOVING THE TEST LABEL");
+    client.pods().inNamespace("ns1").withName("pod2").edit()
+      .editMetadata().removeFromLabels("test").endMetadata()
+      .done();
+
+  //  System.out.println(pod2.getMetadata());
+
+    /*
+    client.pods().inNamespace("ns1").withName("pod2")
+      .patch(new PodBuilder().withNewMetadata()
+        .withLabels(new HashMap<String, String>() {{put("test1", "watch");}})
+        .endMetadata()
+        .build());*/
+
+    System.out.println("Closing watch....");
+    watch.close();
+//    Pod pod3 = new PodBuilder().withNewMetadata().withName("pod3").addToLabels("test", "watch").endMetadata().build();
+
+    System.out.println("Let's end this");
```

Comment:
This looks like WIP

Reply:
Hey, I mistakenly included this file in this commit.. wasn't supposed to be in this PR